web/ui/react-app/src/App.css

```diff
@@ -212,6 +212,42 @@ div.time-input {
     margin-bottom: 20px;
 }
 
+.target-head {
```

Comment:
All styles will become redundant once you start using Reactstrap components

Reply:
custom styling for bold fonts

Reply:
I can see big difference comparing with the old UI
![old_vs_new](https://user-images.githubusercontent.com/10514821/71305982-acb03980-23e3-11ea-981f-22958b2c006b.jpg)

I think you can remove all the custom css without any worries.

Reply:
made the UI almost same to that of old.

Reply:
If we adopt more of the styling of the existing /targets page here (which is a very similar page), we can lose more of these custom styles. E.g. on the targets page the headers are simply `h3` tags, and this should be consistent here. The "show less" / "show more" buttons there do have some custom styling, but it would be good to share that somehow between both pages too.

Reply:
This comment still needs to be addressed as well (more constistency with /new/targets page).